ansible/roles/atmo-ssh-setup/tasks/root-setup.yml

```diff
@@ -0,0 +1,26 @@
+---
+
+- name: Clear the tty from suoers
+  raw: sed -i '/^Defaults\s*requiretty/d' /etc/sudoers
+
+- name: Remove the standard cloud disable command
```

Comment:
minor nitpick, I don't think this task needs to be run when `use_remote_user` is root (i.e. when root SSH login already works). You could add something like `when: 'use_remote_user != "root"'` to effect this